packages/core-data/src/resolvers.js

```diff
@@ -29,14 +29,29 @@ import { ifNotResolved, getNormalizedCommaSeparable } from './utils';
 
 /**
  * Requests authors from the REST API.
+ *
+ * @param {Object|undefined} query Optional object of query parameters to
+ *                                 include with request.
  */
-export function* getAuthors() {
+export function* getAuthors( query ) {
 	const users = yield apiFetch( {
-		path: '/wp/v2/users/?who=authors&per_page=-1',
+		path: addQueryArgs( '/wp/v2/users/?who=authors&per_page=100', query ),
 	} );
 	yield receiveUserQuery( 'authors', users );
 }
 
+/**
+ * Temporary approach to resolving editor access to author queries.
+ *
+ * @param {number} id The author id.
+ */
+export function* __unstableGetAuthor( id ) {
+	const users = yield apiFetch( {
+		path: `/wp/v2/users?who=authors&include=${ id }`,
+	} );
+	yield receiveUserQuery( 'author', users );
```

Comment:
should the first argument here be "authors" to match the same entity as "getAuthors"?

Reply:
Maybe? I think that results in the fetched author overwriting the authors? I can test.

Reply:
Tested changing this and confirmed using the same key results in the query interfering with the `getAuthors` query, causing errors or unexpected results as the results from the two queries overwrite each other.

Reply:
Mmm, so if I'm reading  this right `getAuthor(1)` and  `getAuthor(2)` are using the same query key which means if I trigger both concurrently, they'll mess with each other's result.

Reply:
Yes, I did add dynamic query keys  for both getAuthor and getAuthors, I think it will solve both issues so I reverted the change you did with useMemo